apps/api/src/schema/lib/time.ts

```diff
@@ -53,3 +55,13 @@ export const timeSchema = z
   .regex(TIME_REGEX)
   .openapi({ description: "Time string in 12 or 24 hour format", example: "2:00pm" })
```

Comment:
Consider including an example of 24h time in addition to or in lieu of the 12h time example, since most people probably know what 12h time is.